src/Sylius/Bundle/AdminBundle/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@sylius-ui/admin",
+  "version": "1.0.0",
+  "license": "MIT",
+  "author": "Sylius Sp. z o.o.",
+  "dependencies": {
+    "@popperjs/core": "^2.11.8",
+    "@symfony/webpack-encore": "^3.1.0",
```

Comment:
It's a dev dependency

Reply:
From our perspective it's a regular dependency. `@sylius-ui/admin` is a dev dependency for end-projects.